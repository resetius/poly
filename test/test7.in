^ + * 2147483647 a * 10000000 b 2