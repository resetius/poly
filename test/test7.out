0,2,100000000000000
1,1,42949672940000000
2,0,4611686014132420609